### PR TITLE
MDS-1686 Refactor Mine resource to follow restplus standards

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Im sorry about how large this PR is, it touches a lot of files since the mine object is used in most areas of our code. This will need thorough regression testing and major testing in general.

- adds a new api_model for the mines which defines all the return values for endpoints.
- modified all frontend files to expect a mine_guid instead of a guid.
- added a new resource for getting the mines to display on the map.
- Broke the mine resource into two separate resources
- fixed frontend and backend tests.

### CODE REVIEW

File: frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap [hunk 1]
```diff
@@ -64,15 +64,15 @@ ShallowWrapper {
             Array [
               Object {
                 "label": "New Mine-BLAH6194",
-                "value": "New Mine~fc72863d-83e8-46ba-90f9-87b0ed78823f",
+                "value": "New Mine~undefined",
```
Comment (mmumby) [inline frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap]: The value for the `mine_guid` in this snapshot is being read as `undefined` - double check it is passed into the component correctly, it shouldn't be undefined.

Comment (NWCalvank) [reply frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap]: Meredith with the eagle eyes! 👀

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 2]
```diff
@@ -23,114 +23,119 @@
...
         help='Any additional notes to be added to the mine.',
         trim=True,
-        store_missing=False)
-    parser.add_argument(
-        'tenure_number_id',
-        type=int,
-        help='Tenure number for the mine.',
-        trim=True,
-        store_missing=False)
+        store_missing=False,
```
Comment (Jsyro) [inline python-backend/app/api/mines/mine/resources/mine.py]: I'm starting to waver on the store_missing flag on posts.... without it it will be defaulted to null, which is what the .get() method will do anyways. I feel like i have advocated the other way. but the store_missing is only needed on the put (where the fields supplied actually matter), i think store_missing=False, should be removed from the ListResources.

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 3]
```diff
@@ -332,14 +317,14 @@ def put(self, mine_no_or_guid):
         if 'mine_region' in data:
             mine.mine_region = data['mine_region']
         mine.save()
-
+        current_app.logger.error(f'made it here {tenure}')
```
Comment (Jsyro) [inline python-backend/app/api/mines/mine/resources/mine.py]: debug line.

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 4]
```diff
@@ -332,14 +317,14 @@ def put(self, mine_no_or_guid):
...
         # Tenure validation
         if tenure:
             tenure_exists = MineralTenureXref.find_by_tenure(tenure)
+            current_app.logger.error(f'made it here {tenure} and {tenure_exists}')
```
Comment (Jsyro) [inline python-backend/app/api/mines/mine/resources/mine.py]: same here.

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 5]
```diff
@@ -380,5 +365,61 @@ def get(self):
             mines = Mine.find_by_name_no_permit(search_term)
         else:
             mines = Mine.find_by_mine_name(name_search)
-        result = list(map(lambda x: {**x.json_by_name(), **x.json_by_location()}, mines))
+        result = list(
```
Comment (Jsyro) [inline python-backend/app/api/mines/mine/resources/mine.py]: Are we leaving this one cause it's different? wondering why this doesn't have a model and envelopes etc.

File: python-backend/app/api/mines/status/models/mine_status.py [hunk 6]
```diff
@@ -55,18 +49,27 @@ def create_mine_status_labels_list(self):
...
+    def __repr__(self):
+        return '<MineStatus %r>' % self.mine_status_guid
+
+    def validate_status_code_exists(self, mine_status_xref, mine_status_code, code_or_description):
+        try:
+            return mine_status_xref[mine_status_code][code_or_description]
+        except KeyError:
+            return None
+
+    # def json(self, show_mgr=True):
```
Comment (Jsyro) [inline python-backend/app/api/mines/status/models/mine_status.py]: remove commented code?

File: python-backend/app/api/mines/namespace/mines.py [hunk 7]
```diff
@@ -21,7 +22,10 @@
 
 api = Namespace('mines', description='Mine related operations')
 
-api.add_resource(MineResource, '', '/<string:mine_no_or_guid>')
+api.add_resource(MineResource, '/<string:mine_no_or_guid>')
+api.add_resource(MineListResource, '')
+api.add_resource(MineMapResource, '/map-list')
```
Comment (NWCalvank) [inline python-backend/app/api/mines/namespace/mines.py]: I don't think this is an appropriate endpoint for our API to have. The API knows nothing about the map on the frontend. It's probably more reasonable for this to be something like `(MineBasicInfoResource, '/basic')`

Not sure if we're already referring to something as `BasicInfo` elsewhere. I'm not committed to that name if we are -- but I think it serves the purpose of illustrating my point

File: python-backend/app/api/applications/resources/application.py [hunk 8]
```diff
@@ -22,24 +22,35 @@
 class ApplicationListResource(Resource, UserMixin):
     parser = reqparse.RequestParser()
     parser.add_argument(
-        'application_no', type=str, required=True, help='Number of the application being added.')
+        'application_no',
+        type=str,
+        required=True,
+        help='Number of the application being added.',
+        location='form')
```
Comment (NWCalvank) [inline python-backend/app/api/applications/resources/application.py]: What's the story behind these application changes? Seems unrelated to the rest of the PR

File: python-backend/app/api/documents/expected/resources/expected_document_uploads.py [hunk 9]
```diff
@@ -80,7 +80,7 @@ def put(self, expected_document_guid):
             if not mine_doc:
                 return self.create_error_payload(404, 'Mine Document not found'), 404
 
-            expected_document.mine_documents.append(mine_doc)
+            expected_document.related_documents.append(mine_doc)
```
Comment (NWCalvank) [inline python-backend/app/api/documents/expected/resources/expected_document_uploads.py]: Not sure why expected_document is being updated in this PR, but the line(s) below should be updated to use `save()` as per a recent chat with @Jsyro

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 10]
```diff
@@ -23,114 +23,119 @@
...
+            'current_page': pagination_details.page_number,
+            'total_pages': pagination_details.num_pages,
+            'items_per_page': pagination_details.page_size,
+            'total': pagination_details.total_results,
+        }
+
+    @api.expect(parser)
+    @api.doc(
+        description=
+        'This endpoint creates a new mine with the information provided and returns it. If an error occurs the appropriate response is returned.'
```
Comment (NWCalvank) [inline python-backend/app/api/mines/mine/resources/mine.py]: I'd like us to avoid using "This endpoint" at the start of every description. An active, user-action-focused voice is much more direct and concise. "Create a new mine."

The "with the information provided" is implied by the nature of a REST API, imo. "and returns it" should be obvious from the rest of the documentation for the endpoint.

And I think it's reasonable for someone to expect an error to be returned if an error occurs. Adding that to the description is unnecessarily wordy

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 11]
```diff
@@ -23,114 +23,119 @@
...
-            'mine_no_or_guid':
-            'Mine number or guid. If not provided a paginated list of mines will be returned.'
-        })
+            'per_page': 'The number of results to be returned per page.',
+            'page': 'the current page number to be displayed.',
+            'search': 'The search term.',
+            'commodity': 'A commodity to filter the mine list by.',
+            'status': 'A mine status to filter the mine list by.',
+            'tenure': 'A mine tenure type to filter the mine list by.',
+            'region': 'A mine region to filter the mine list by.',
```
Comment (NWCalvank) [inline python-backend/app/api/mines/mine/resources/mine.py]: The grammar nerd in me is really bothered by the prepositions at the end of these statements

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 12]
```diff
@@ -23,114 +23,119 @@
...
+        required=True,
+        location='json')
 
     @api.doc(
         params={
-            'mine_no_or_guid':
-            'Mine number or guid. If not provided a paginated list of mines will be returned.'
-        })
+            'per_page': 'The number of results to be returned per page.',
+            'page': 'the current page number to be displayed.',
```
Comment (NWCalvank) [inline python-backend/app/api/mines/mine/resources/mine.py]: Missing capitalization

File: python-backend/app/api/mines/mine/resources/mine.py [hunk 13]
```diff
@@ -23,114 +23,119 @@
...
+            'search': 'The search term.',
+            'commodity': 'A commodity to filter the mine list by.',
+            'status': 'A mine status to filter the mine list by.',
+            'tenure': 'A mine tenure type to filter the mine list by.',
+            'region': 'A mine region to filter the mine list by.',
+            'major': 'True or false, filters the mine list by major mines or regional mines.',
+            'tsf': 'True or false, filter the mine list by mines with or without a TSF.',
```
Comment (NWCalvank) [inline python-backend/app/api/mines/mine/resources/mine.py]: We don't mention the data type in any of the other params here. Is there a reason why we'd say "True or false?" And if we do, we should probably say "Boolean."

File: python-backend/app/api/mines/mine/resources/mine_map.py [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+        # Below caches the mine map response object in redis with a timeout.
+        # Generating and jsonifying the map data takes 4-7 seconds with 50,000 points,
+        # so caching seems justified.
+        #
+        # TODO: Use some custom representation of this data vs JSON. The
+        # json string is massive (with 50,000 points: 16mb uncompressed, 2.5mb compressed).
+        # A quick test using delimented data brings this down to ~1mb compressed.
+        map_result = cache.get(MINE_MAP_CACHE)
+        last_modified = cache.get(MINE_MAP_CACHE + '_LAST_MODIFIED')
+        if not map_result:
```
Comment (NWCalvank) [inline python-backend/app/api/mines/mine/resources/mine_map.py]: Flagging this for later. I think this piece of the API knows far too much about how it's being consumed. It's location data, but there's nothing (from what I can tell) that forces it to be used by a map.

This should probably be more fully decoupled from the frontend use case at some point. Thoughts @Jsyro @nayr974 ?

File: python-backend/app/api/mines/status/models/mine_status.py [hunk 15]
```diff
@@ -55,18 +49,27 @@ def create_mine_status_labels_list(self):
...
+    #     status_labels_list = self.create_mine_status_labels_list()
+    #     return {
+    #         'mine_status_guid': str(self.mine_status_guid),
+    #         'mine_guid': str(self.mine_guid),
+    #         'mine_status_xref_guid': str(self.mine_status_xref.mine_status_xref_guid),
+    #         'status_values': status_values_list,
+    #         'status_labels': status_labels_list,
+    #         'effective_date': self.effective_date.isoformat(),
+    #         'expiry_date': self.expiry_date.isoformat()
+    #     }
```
Comment (NWCalvank) [inline python-backend/app/api/mines/status/models/mine_status.py]: Comment to remove? Why would the `mine_status` json function be changed/removed here?

File: python-backend/tests/auth/test_expected_auth.py [hunk 16]
```diff
@@ -50,34 +51,50 @@
...
+     (RequiredDocumentResource, "get", [MINE_VIEW]),
      (PermitAmendmentResource, "post", [MINE_CREATE]),
      (PermitAmendmentResource, "put", [MINE_CREATE]),
      (PermitAmendmentResource, "delete", [MINE_ADMIN]),
      (PermitAmendmentDocumentResource, "post", [MINE_CREATE]),
      (PermitAmendmentDocumentResource, "put", [MINE_CREATE]),
      (PermitAmendmentDocumentResource, "delete", [MINE_CREATE]),
-     (RequiredDocumentResource, "get", [MINE_VIEW]), (MinespaceUserResource, 'get', [MINE_ADMIN]),
-     (MinespaceUserResource, 'post', [MINE_ADMIN]), (MinespaceUserResource, 'delete', [MINE_ADMIN]),
+     (RequiredDocumentResource, "get", [MINE_VIEW]),
```
Comment (NWCalvank) [inline python-backend/tests/auth/test_expected_auth.py]: This doesn't look quite alphabetical.

File: frontend/src/components/mine/MineDashboard.js [hunk 17]
```diff
@@ -123,7 +125,8 @@ export class MineDashboard extends Component {
 
   loadMineData(id) {
     this.props.fetchMineRecordById(id).then(() => {
-      this.props.fetchApplications({ mine_guid: this.props.mines[id].guid });
+      this.props.fetchApplications({ mine_guid: this.props.mines[id].mine_guid });
+      this.props.fetchPermits({ mine_guid: this.props.mines[id].mine_guid });
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply frontend/src/components/mine/MineDashboard.js]: We were fetching the permits on there own for the permits tab so I brought it out to the dashboard so that the permits could be fetched for the summary tab and remove most of theyre information from the mine object.

File: frontend/src/components/dashboard/minesHomePage/Dashboard.js [hunk 18]
```diff
@@ -156,9 +161,15 @@ export class Dashboard extends Component {
...
-    this.props.fetchMineRecords(params).then(() => {
-      this.setState({ mineList: true });
-    });
+    if (remainingParams.map) {
+      this.props.fetchMineRecordsForMap().then(() => {
+        this.setState({ mineList: true });
+      });
+    } else {
+      this.props.fetchMineRecords(params).then(() => {
+        this.setState({ mineList: true });
```
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/components/dashboard/minesHomePage/Dashboard.js]: Good catch! 
From what I can see, the `map=true` param is still useful, just in a different way than it was being used before. (ie frontend check vs backend) I can see how it will be confusing going forward. 

Better approaches:
Remove that param from the url, only use the `this.fetchDataFromUrl` to update the mineList using `this.props.fetchMineRecords`

`this.props.fetchMineRecordsForMap` could be run in the background on `componentDidMount()`, and stored in a different const in the mineReducer, - The benefit here, is that the user will not have to wait for all the mine information to be fetched when they navigate to the map view.

File: frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap (hunk 1 above)
Comment (JustinMacaulay) [reply frontend/src/tests/components/admin/__snapshots__/NewMinespaceUser.spec.js.snap]: Missed one part of the mock data update. Fixed and good catch!

File: frontend/src/tests/reducers/minespaceReducer.spec.js [hunk 19]
```diff
@@ -51,7 +51,7 @@ describe("mineReducer", () => {
         mine_status: [],
         mine_tailings_storage_facility: [],
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply frontend/src/tests/reducers/minespaceReducer.spec.js]: Yes and changed.

File: python-backend/app/api/applications/resources/application.py (hunk 8 above)
Comment (JustinMacaulay) [reply python-backend/app/api/applications/resources/application.py]: This was me updating the previous endpoints I had worked on to bring them inline with some new information we discovered.

File: frontend/src/actionCreators/mineActionCreator.js [hunk 20]
```diff
@@ -179,6 +179,21 @@ export const fetchMineRecords = (params) => (dispatch) => {
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchMineRecordsForMap = () => (dispatch) => {
+  dispatch(request(reducerTypes.GET_MINE_RECORDS));
```
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/actionCreators/mineActionCreator.js]: correct, it _should_ be different.

However, it appears he is using the same action as `fetchMineRecords` and the data is being kept in the same spot. So in this situation, the reducer being the same is fine. 


I did recommend in a comment on `Dashboard.js` to make this its own action and store the data in a different const than the `mineReducer`, the benefit being that this can be fitched when the component mounts vs when the user navigates to the tab thus delaying loading time. If that switch is made, then this would need to have its own reducerType and actionType.

File: python-backend/app/api/documents/expected/resources/expected_document_uploads.py (hunk 9 above)
Comment (JustinMacaulay) [reply python-backend/app/api/documents/expected/resources/expected_document_uploads.py]: A lot of these random changes are due to the fact that what the frontend is expecting is named differently than what is on the models. I had to change around a lot of references in other classes due to this.

File: python-backend/app/api/mines/mine/resources/mine.py (hunk 2 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine.py]: True. I will remove them from the parser for the posts.

File: python-backend/app/api/mines/mine/resources/mine.py (hunk 3 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine.py]: removed.

File: python-backend/app/api/mines/mine/resources/mine.py (hunk 4 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine.py]: Also removed.

File: python-backend/app/api/mines/mine/resources/mine.py (hunk 5 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine.py]: This one is different as it is returning an object with mixed values from several models. I tried to get it to work with response marshalling but was unable.

File: python-backend/app/api/mines/mine/resources/mine_map.py [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+    'longitude': fields.String,
+})
+
+mine_map_list = api.model(
+    'MineMap', {
+        'mine_guid': fields.String,
+        'mine_name': fields.String,
+        'mine_no': fields.String,
+        'mine_location': fields.Nested(mine_location)
+    })
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine_map.py]: Forgot to move these ones. Thanks!

File: python-backend/app/api/mines/mine_api_models.py [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
+        'verified_status': fields.Nested(MINE_VERIFIED)
+    })
+
+MINE = api.inherit(
+    'Mine', MINES, {
+        'mine_location': fields.Nested(MINE_LOCATION),
+        'mine_expected_documents': fields.List(fields.Nested(MINE_EXPECTED_DOCUMENT)),
+    })
+
+MINE_LIST_MODEL = api.model(
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine_api_models.py]: I added model to all of them and moved all the models to the constants model file.

File: python-backend/tests/mines/resources/test_mine_list_resource.py [hunk 23]
```diff
@@ -6,39 +6,28 @@ def test_get_mines(test_client, auth_headers):
     get_resp = test_client.get('/mines', headers=auth_headers['full_auth_header'])
     get_data = json.loads(get_resp.data.decode())
     assert get_resp.status_code == 200
-    assert get_data['mines'][0]['guid'] == TEST_MINE_GUID
+    assert get_data['mines'][0]['mine_guid'] == TEST_MINE_GUID
 
 
 def test_get_mines_with_map_true(test_client, auth_headers):
     get_resp = test_client.get('/mines?map=true', headers=auth_headers['full_auth_header'])
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply python-backend/tests/mines/resources/test_mine_list_resource.py]: Removed this test.

File: python-backend/app/api/mines/mine/resources/mine_map.py (hunk 14 above)
Comment (nayr974) [reply python-backend/app/api/mines/mine/resources/mine_map.py]: Primarily naming and comments at this point, correct?

Comment (NWCalvank) [reply python-backend/app/api/mines/mine/resources/mine_map.py]: Yeah, I don't think it's any functional changes at this point, which is good -- but generalizing the naming could help us steer clear of adding map-specific functionality in the future.

File: frontend/src/components/mine/MineDashboard.js (hunk 17 above)
[the remark this answers is not in the document]
Comment (NWCalvank) [reply frontend/src/components/mine/MineDashboard.js]: Cool. Marking as resolved -- we can carefully confirm in manual testing that this doesn't prevent permits from being loaded on any views

File: python-backend/app/api/applications/resources/application.py (hunk 8 above)
Comment (NWCalvank) [reply python-backend/app/api/applications/resources/application.py]: Okay. I'm marking it as resolved. With a mammoth PR like this, it's probably best to split out these changes if they're not 100% related to the primary PR goal

File: frontend/src/actionCreators/mineActionCreator.js [hunk 24]
```diff
@@ -179,6 +179,21 @@ export const fetchMineRecords = (params) => (dispatch) => {
...
+  dispatch(showLoading());
+  return CustomAxios()
+    .get(ENVIRONMENT.apiUrl + API.MINE_MAP_LIST, createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_MINE_RECORDS));
+      dispatch(mineActions.storeMineList(response.data));
+      dispatch(hideLoading());
+      return response;
+    })
+    .catch(() => dispatch(error(reducerTypes.GET_MINE_RECORD)))
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply frontend/src/actionCreators/mineActionCreator.js]: Its also that way in the fetchMineRecords which I based this action creator off of. Should that also be changed?

File: python-backend/app/api/mines/mine/resources/mine.py (hunk 13 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine/resources/mine.py]: I have modified these descriptions.

File: python-backend/app/api/mines/status/models/mine_status.py (hunk 15 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/status/models/mine_status.py]: If its no longer used by any returns. I am not sure about that though so I have it commented out for now and would like some more rigorous testing.

Comment (NWCalvank) [reply python-backend/app/api/mines/status/models/mine_status.py]: If the intention is to remove it (assuming that nothing blows up in manual testing) then we can just remove it. We have a git history of it if we ever need to get it back

File: frontend/src/actionCreators/mineActionCreator.js (hunk 24 above)
[the remark this answers is not in the document]
Comment (NWCalvank) [reply frontend/src/actionCreators/mineActionCreator.js]: I'll defer to @mmumby on that. If we haven't had to update `fetchMineRecords` then this probably doesn't cause any real problems, but I would expect the `error` to match the `success`

File: python-backend/app/api/mines/mine_api_models.py [hunk 25]
```diff
@@ -0,0 +1,117 @@
...
+    'MineList', {
+        'mines': fields.List(fields.Nested(MINES)),
+        'current_page': fields.Integer,
+        'total_pages': fields.Integer,
+        'items_per_page': fields.Integer,
+        'total': fields.Integer,
+    })
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply python-backend/app/api/mines/mine_api_models.py]: Yes, this is 100% a way you could handle this. I only ran into this return in one place though.

File: frontend/package-lock.json [hunk 26]
```diff
@@ -1191,6 +1191,7 @@
       "version": "2.6.1",
       "resolved": "https://registry.npmjs.org/async/-/async-2.6.1.tgz",
       "integrity": "sha512-fNEiL2+AZt6AlAw/29Cr0UDe4sRAHCpEHh54WMz+Bb7QfNcFw4h3loofyJpLeQs4Yx7yuqu/2dLgM5hKOs6HlQ==",
+      "optional": true,
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply frontend/package-lock.json]: I ran NPM CI so I'm not sure where this came from.

File: python-backend/tests/auth/test_expected_auth.py (hunk 16 above)
Comment (JustinMacaulay) [reply python-backend/tests/auth/test_expected_auth.py]: I put it in alphabetical order now.

File: python-backend/app/api/mines/namespace/mines.py (hunk 7 above)
Comment (nayr974) [reply python-backend/app/api/mines/namespace/mines.py]: mines/coordinates?

File: frontend/src/actionCreators/mineActionCreator.js (hunk 24 above)
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/actionCreators/mineActionCreator.js]: Nathan is correct. 

It doesn't break anything, This is used more for logging purposes, but the request, success, and error should all be the same ReducerType.

File: python-backend/app/api/mines/mine_api_models.py (hunk 25 above)
[the remark this answers is not in the document]
Comment (NWCalvank) [reply python-backend/app/api/mines/mine_api_models.py]: Nice. I'll mark this as resolved since it's not direct feedback for this PR. I think we could benefit from this pattern in the future, though. Thanks, Justin

File: frontend/package-lock.json (hunk 26 above)
[the remark this answers is not in the document]
Comment (NWCalvank) [reply frontend/package-lock.json]: Very strange. Maybe you have a different version of `npm`? This shouldn't be a big deal, so I'll mark this as resolved. I just wanted to highlight this in case you were using the wrong command

File: python-backend/app/api/mines/namespace/mines.py (hunk 7 above)
Comment (NWCalvank) [reply python-backend/app/api/mines/namespace/mines.py]: I'd like to see this changed, but I might just split it into a separate tech debt ticket that deals with removing references to the map from the backend since those references exist in a few places

File: python-backend/app/api/mines/status/models/mine_status.py (hunk 6 above)
Comment (JustinMacaulay) [reply python-backend/app/api/mines/status/models/mine_status.py]: Un-commented as this function was used in other resources.
